### ISSUE TITLE
ctrl-f ctrl-b and friends are very slow

### ISSUE THREAD

**Issue by bronson** [atom/vim-mode#668]
It feels like Ctrl-F waits 500ms before responding.  This makes vim-mode feel really slow, especially since I hit this key combo hundreds of times per day. (This is also true of Ctrl-B, Ctrl-D, Ctrl-U, etc.)

PageDown responds instantly.  I know it's possible.

I tried disabling the native ctrl-f bindings but it's still slow:

```cson
'.platform-darwin .tree-view':
  'ctrl-f': 'unset!'
'body':
  'ctrl-f': 'unset!'
'body .native-key-bindings':
  'ctrl-f': 'unset!'
```

Also tried setting ctrl-f to fire pgdown.  Still slow:

```cson
'body':
  'ctrl-f': 'core:page-down'
'body .native-key-bindings':
  'ctrl-f': 'core:page-down'
```

How can I make ctrl-f respond instantly like PageDown?

This is seen using 0.199.0 on Mac and Arch Linux.

**Comment by FernandoBasso** [comment]
Yes, I confirm this on Arch Linux as well with Atom 0.199.9.

**Comment by dcalhoun** [comment]
I've noticed this as well with the Atom 0.199.9 release. It was not like this with the previous release.

**Comment by maxbrunsfeld** [comment]
The slowness was due to lots of stack traces being created because deprecated APIs were being used. Fixed via #630.
